output/transform/postgres.go

```diff
@@ -93,6 +94,19 @@ func transformPostgresDatabases(s snapshot.FullSnapshot, transientState state.Tr
 		}
 
 		s.DatabaseInformations = append(s.DatabaseInformations, &info)
+
+		diff, exist := diffState.DatabaseStats[database.Oid]
+		if exist {
+			stat := snapshot.DatabaseStatistic{
+				DatabaseIdx:        databaseOidToIdx[database.Oid],
+				FrozenXidAge:       database.FrozenXIDAge,
+				MinMxidAge:         database.MinMXIDAge,
+				XactCommit:         database.XactCommit,
+				XactRollback:       database.XactRollback,
```

Comment:
I think these also need to come out of the `diff.`, similar to TransactionsPerSecond.